design/baremetal-operator/kubebuilder-migration.md

```diff
@@ -0,0 +1,202 @@
+# kubebuilder-migration
+
+## Status
+
+implementable
+
+## Summary
+
+We have recently encountered several situations where we wanted to
+take advantage of kubernetes API features that are not supported by
+the old version of operator-sdk currently used with the
+baremetal-operator.  The newer version of operator-sdk is more closely
```

Comment:
Specifically, versions after 1.0 I think.

Reply:
I added a note to that effect.